tests/test_timedcall.py

```diff
@@ -10,13 +10,14 @@
 # limitations under the License.
 
 import multiprocessing
+import sys
 import time
 import unittest
 
 from tests.utils import TimedCall
 
 
-@TimedCall(seconds=10, force_quit=False)
+@TimedCall(seconds=10 if sys.platform == "linux" else 60, force_quit=False)
```

Comment:
a 1s test could take up to 60s if not on linux?

Reply:
this happens from time to time with github's macOS instances...

Reply:
ok